### DISCUSSION TITLE
Decoding texts from specific sources

### DISCUSSION THREAD

**Discussion by Eran-T** [freelawproject/courtlistener#3372] [Q&A] [answered]
Hi,
I have a question regarding a specific opinion I ran into. (https://www.courtlistener.com/api/rest/v3/opinions/4560099/)
It seems that while extracting the contents of the opinion (seems to have went through OCR) the content was either decoded incorrectly or noise was added (as there does seem to be a proper utf-8/unicode section in there).

My question is whether you might have an assumption as to how it ended up this way and whether the non-coherent section is decodable. 

Thanks for the help!

**Comment by mlissner** [comment]
Woo, that's a weird one. On the last page, there's an image, so that is probably what triggered our OCR engine, but I don't know why the rest is entirely wrong. 

I've seen something like this before with fonts that are embedded into PDFs, where the font file was corrupt. 

@flooie can you look into this, fix this doc, and see if there are others that seem to suffer the same fate?

**Reply by flooie** [reply]
This is very strange. This is a Tennessee court upload form - but the PDF is very strangely structured.

**Comment by flooie** [comment, marked as answer]
@Eran-T Thanks for spotlighting this.  I found 12 corrupted PDFs.  The court clerk was kind enough to generate new opinions with corrected PDF files and they have all been uploaded into the system correctly.

**Reply by mlissner** [reply]
Nicely done!
